pip/rules.bzl

```diff
@@ -317,6 +326,14 @@ deploy_pip = rule(
             default = "",
             doc = "Repository name in the .pypirc profile to deploy to"
         ),
+        "distribution_tag": attr.string(
```

Comment:
Let's check what fails if we use an invalid tag - if it fails in PIP that's fine